Resources/Prototypes/XenoArch/Effects/utility_effects.yml

```diff
@@ -247,3 +247,18 @@
     - behavior: Pulsing
       changeSound:
         path: /Audio/Items/change_drill.ogg
+
+- type: artifactEffect
+  id: EffectDash
+  targetDepth: 3
+  effectHint: artifact-effect-hint-displacement
+  whitelist:
+    components:
+    - Item
+  permanentComponents:
+  - type: DashAbility
```

Comment:
fairly sure this requires ninja to be used?

Reply:
it doesn't, I tested